src/components/Link.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Link as USWDSLink } from "@trussworks/react-uswds";
+import { Link as GatsbyLink } from "gatsby";
+
+// Pass only activeClassName and partiallyActive to GatsbyLink
+// Pass only fileType and size to file downloads
+const Link = ({
+  children,
+  to,
+  activeClassName,
+  partiallyActive,
+  variant,
+  fileType,
+  size,
+  ...other
+}) => {
+  // Assumes that any internal link will start with exactly one slash, and that anything else is external.
+  const internal = /^\/(?!\/)/.test(to);
+
+  // Assumed file download link structure
```

Comment:
What is the assumed file download link structure?

Reply:
Good question! I took it directly from the Gatsby documentation. But from my understanding, the file download link structure satisfies with a period, in the beginning, followed by 1 or more numbers or letters, and then the end of the line is represented by `$`. The `/i` at the end ignores the case of the string (case-insensitive).